azure-pipelines.yml

```diff
@@ -296,4 +296,18 @@ stages:
     parameters:
       # Symbol validation isn't being very reliable lately. This should be enabled back
       # once this issue is resolved: https://github.com/dotnet/arcade/issues/2871
-      enableSymbolValidation: false
\ No newline at end of file
+      enableSymbolValidation: false
+      # This is to enable SDL runs part of Post-Build Validation Stage
+      SDLValidationParameters:
+        enable: true
+        continueOnError: false
+        params: ' -SourceToolsList @("policheck","credscan")
+        -TsaInstanceURL $(_TsaInstanceURL)
```

Comment:
I  believe these variables for Arcade are coming from a variable group that is referenced here 
https://github.com/dotnet/arcade/blob/e2e223ceb1a140788fa8e48c6b90ad6bb5064419/eng/common-variables.yml#L31

Reply:
Looks like I need to add https://github.com/dotnet/winforms/blob/15eb866a9db9a5e9856ad8072e83acdd778770ee/azure-pipelines.yml#L17-L19

Reply:
Just make sure that all the values match. I think the expectation is that repos have their own variable groups so that say, aspnetcore bugs don't get filed under the winforms area path.